en/04_Changelogs/5.0.0.md

```diff
@@ -102,6 +126,25 @@ This is a major release and as a result there are a number of breaking API chang
 - `SilverStripe\Core\Extensible::invokeWithExtensions()` and `SilverStripe\Core\Extensible::extend()` now use the splat operator instead of having a concrete number of possible arguments.
 - The constructor for `SilverStripe\Versioned\Versioned` now explicitly only accepts mode as a single argument.
 
+### Email {#api-email}
+
+- Email was sent in CMS 4 using [SwiftMailer](https://swiftmailer.symfony.com/docs/introduction.html), which has since [gone End Of Life](https://symfony.com/blog/the-end-of-swiftmailer). In CMS 5 this has been replaced with [symfony/mailer](https://symfony.com/doc/current/mailer.html). `symfony/mailer` is the currently maintained email package from Symfony. It's a more flexible email system that allows easier integration with third-party email providers.
+- In CMS 4 the `SilverStripe\Control\Email\Email` class subclassed `SilverStripe\View\ViewableData`. In CMS 5 it now subclasses `Symfony\Component\Mime\Email`.
```

Comment:
```suggestion
- In CMS 4 the [Email](api:SilverStripe\Control\Email\Email) class subclassed [ViewableData](api:SilverStripe\View\ViewableData). In CMS 5 it now subclasses `Symfony\Component\Mime\Email`.
```

Reply:
Rather than play peer review ping pong to update every Silverstripe `FQCN` to an api link, I think it'll be far more efficient to simply do this in one big go when we finalize the 5.0.0 changelog prior to releasing 5.0.0.

There's currently loads of other Silverstripe `FQCN` in the changelog unrelated to email that aren't api links